mne/realtime/fieldtrip_client.py

```diff
@@ -171,6 +171,8 @@ def _guess_measurement_info(self):
                     this_info['kind'] = FIFF.FIFFV_ECG_CH
                 elif ch.startswith('MISC'):
                     this_info['kind'] = FIFF.FIFFV_MISC_CH
+                elif ch.startswith('SYS'):
+                    this_info['kind'] = FIFF.FIFFV_SYST_CH
```

Comment:
Yep, but there's no way to know that short_recording matches the current dacq parameters, right?